cmake/Testing.cmake

```diff
@@ -34,15 +34,16 @@ function(auto_test subsystem module extra_deps)
     COMMAND ${TEST_CROSSCOMPILING_EMULATOR} test_${module})
 endfunction()
 
-auto_test(core core ${${PROJECT_NAME}_RESOURCES})
+auto_test(core core "${${PROJECT_NAME}_RESOURCES}")
 auto_test(core contactid "")
 auto_test(core toxid "")
 auto_test(core toxstring "")
 auto_test(chatlog textformatter "")
-auto_test(net bsu ${${PROJECT_NAME}_RESOURCES}) # needs nodes list
+auto_test(net bsu "${${PROJECT_NAME}_RESOURCES}") # needs nodes list
 auto_test(persistence paths "")
 auto_test(persistence dbschema "")
 auto_test(persistence offlinemsgengine "")
+auto_test(persistence smileypack "${${PROJECT_NAME}_RESOURCES}") # need smilies qrc
```

Comment:
Note that I found an existing bug in this file: when you pass a list into a cmake function, if you didn't quotate ("") it, it will only take the first entry. This was not noticeable until now since we only need the first entry every time.